### PR TITLE
4.9 added pulp token - LP erc-20 - building tests

### PR DESCRIPTION
**Pull request by Alexangelj**
Added ERC-20 inheritance to Pool.sol contract. Added appropriate conversions between Ether, Pool Token, and cEther token.

### CODE REVIEW

File: test/protocol.js [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+            "a < b"
+        );
+        /* Check for ERC20 interface parameter */
+
+        /* Assert liability of msg sender added the xis */
+
+        /* Assert returns the nonce of the token */
+    });
+
+    it('should be able to withdraw funds', async () => {
```
Comment (BrockSmedley) [inline test/protocol.js]: Why do we have so many of the same test here? And why are they empty?

File: test/poolV2.js [hunk 2]
```diff
@@ -173,4 +172,22 @@ contract('Pool', accounts => {
...
+    it('should be able to exercise a Prime', async () => {
+        let tokenId = await _prime.nonce();
+
+        /* Bob then exercises the Prime */
+        let collateralAmount = await web3.utils.toWei('0.1');
+        let strikeAmt = await web3.utils.toWei('1');
+        await _tUSD.approve(_prime.address, strikeAmt, {from: Bob});
+        await _tUSD.mint(Bob, strikeAmt);
+
+        console.log('Exercises ether denominated prime');
```
Comment (BrockSmedley) [inline test/poolV2.js]: we should probably start deleting console.log statements. They add up and get messy pretty quick. If we want handmade logs, we should implement a more robust logging system like [pino](https://getpino.io/#/).

File: test/protocol.js (hunk 1 above)
Comment (Alexangelj) [reply test/protocol.js]: was building this while we were talking about prs so I just pushed all my changes so I can start making prs

File: test/poolV2.js (hunk 2 above)
Comment (Alexangelj) [reply test/poolV2.js]: I'll be doing assert statements and no console logs. I was basically acting as a human assert by checking the logs

**Pull request merged**